modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -31,6 +31,24 @@ module LocaleModel {
   use LocaleModelHelpFlat;
   use LocaleModelHelpMem;
 
+  //
+  // The task layer calls these to convert between full sublocales and
+  // execution sublocales.  Full sublocales may contain more information
+  // in some locale models, but not in this one.
+  //
+  export
+  proc chpl_localeModel_sublocToExecutionSubloc(full_subloc:chpl_sublocID_t)
```

Comment:
The `chpl_localeModel_` prefix here is probably not needed.  We need this sort of thing for runtime symbols because they're in C and the namespace is flat, but in Chapel code we could disambiguate using 'module.' syntax in the unlikely event `sublocToExecutionSubloc` was ambiguous.  And the shorter names are nice.